hosted_control_planes/hcp-deploy/hcp-deploy-ibmz.adoc

```diff
@@ -66,6 +66,14 @@ include::modules/hcp-ibmz-zvm-agents.adoc[leveloffset=+2]
 
 include::modules/hcp-ibmz-scale-np.adoc[leveloffset=+1]
 
+include::modules/disconnected-install-ibmz-hcp.adoc[leveloffset=+1]
```

Comment:
order of the topics are wrong , refer box note for the correct order. It should be :
1. Adding Credentials and Registry Certificate Authority to the Management Cluster
2. Update Registry Certificate Authority with Private Registry
3. Adding the Registry Certificate Authority to the Hosted Cluster